engines/griffon/cutscenes.cpp

```diff
@@ -207,6 +234,8 @@ void GriffonEngine::intro() {
 	int cnt = 0;
 	float xofs = 0.0;
 	float ld = 0.0;
+	int nextparagraph = 10;
```

Comment:
Don't we want to read the first 10 lines as well before the actual start of the story?